docs/CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+3.0.1-SNAPSHOT
```

Comment:
Can you move your updated into `3.0.0-SNAPSHOT`, please?

Reply:
Oh, sure.